Jenkinsfile

```diff
@@ -17,11 +17,12 @@ pipeline {
       }
       steps {
 	sh 'python3 -c "import torch; torch.cuda.current_device()"'
+	sh 'python3 -c "import tensorflow as tf; tf.test.is_gpu_available()"'
 	sh 'python3 -m venv --system-site-packages --without-pip $HOME'
 	sh '''#!/bin/bash -ex
 	  source $HOME/bin/activate
 	  python3 setup.py develop
-	  KYMATIO_BACKEND=$STAGE_NAME python3 -m pytest --cov=kymatio
```

Comment:
At this point, the old backends have not been removed, so we need the environment variables for the tests to work.

Reply:
Right.